congressional-support.yaml

```diff
@@ -0,0 +1,119 @@
+- type: house
+  name: House Documents Repository
+  url: http://docs.house.gov/
+
+- type: house
+  name: Office of Compliance
+  url: http://www.compliance.gov/
+
+- type: house
+  name: Clerk of the House
+  url: http://clerk.house.gov/
+
+- type: house
+  name: Chief Administrative Officer
+  url: http://cao.house.gov/
+
+- type: house
+  name: Sergeant at Arms
+  url:
+
+- type: house
+  name: Chaplain
+  url: http://chaplain.house.gov/
+
+- type: house
+  name: Inspector General
+  url: http://www.house.gov/content/learn/officers_and_organizations/inspector_general.php/
```

Comment:
Looks like the process you used adds a trailing slash onto each URL. That's not preferable, especially for an extended URL like this. Can you remove the trailing slashes?